sleap/gui/commands.py

```diff
@@ -363,10 +363,12 @@ def exportFullPackage(self):
     def previousLabeledFrame(self):
         """Goes to labeled frame prior to current frame."""
         self.execute(GoPreviousLabeledFrame)
+        self.state["instance"] = None
```

Comment:
We want this logic to go inside the class that is being executed (i.e. `GoPreviousLabeledFrame`).

```suggestion
```